engine/univNames.ml

```diff
@@ -65,10 +66,27 @@ let subst_ubinder (subst,(ref,l as orig)) =
   let ref' = fst (Globnames.subst_global subst ref) in
   if ref == ref' then orig else ref', l
 
+let name_universe lvl =
+  (** Best-effort naming from the string representation of the level. This is
+      completely hackish and should be solved in upper layers instead. *)
+  try Name (Id.of_string_soft (Level.to_string lvl)) with _ -> Anonymous
```

Comment:
What exception can be raised here?

Reply:
If some code generates universes with invalid unicode I think it deserves to explode.